src/main/java/com/cloudogu/wiki/Wiki.java

```diff
@@ -40,9 +42,22 @@
     public Wiki(String repositoryId, String branchName, String displayName, String description, String revision) {
         this.repositoryId = repositoryId;
         this.branchName = branchName;
-        this.displayName = displayName;
         this.description = description;
         this.revision = revision;
+
+        // displayName has structure 'group/name' -> splitting it
+        // group can have the structure 'name1/name2/...'
+        String splitedName[] = displayName.split(SEPARATOR, 2);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the array designator from the variable to the type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1197)